CDCgov/prime-reportstream
# Support for windows users

This PR adds support for Windows base systems so that RS can run in a Windows environment.

Test Steps:
1. Be on Windows or Unix base System
2. Use Git Bash (Windows) or similar tool for your terminal
3. Run ./gradlew clean package without anything failing.
4. Open project in intelliJ or IDE of choice
5. Reset indexes / invalidate caches
6. Make sure all dependencies load and that it pulls Kotlin 1.9.23
7. Run ./graldew clean package from ide terminal and make sure everything passes

## Changes
- `.gitattributes`: Let git handle auto text formatting across different platforms
- `buildSrc/build.gradle.kts`: Get Kotlin version
- `prime-router/build.gradle`: Add functionality to `killFunc` task to run in Windows or Unix-base systems
- `prime-router/src/test/kotlin/cli/ValidateYMLCommandTests.kt`: Enhance unit test so it runs on Windows or on a Unix-base system

## Checklist

### Testing
